config.py

```diff
@@ -212,6 +212,7 @@ def get_database_uri(name, host, censored=True):
 
 AT_USERNAME = specific_parser["AFRICASTALKING"]["username"]
 AT_API_KEY = specific_parser["AFRICASTALKING"]["api_key"]
+AT_SENDER_ID = specific_parser["AFRICASTALKING"].get("at_sender_id")
```

Comment:
@enjeyw No config file change?

Reply:
config file change had already snuck in lol

Reply:
haha no worries then

Reply:
was wondering why tests didn't fail on that

Reply:
i mean they shouldn't fail anyways because it's a `get`